pkg/testutils/utils.go

```diff
@@ -1,4 +1,4 @@
-//nolint:wrapcheck
+//nolint:wrapcheck // errors do not need to be wrapped in unit and integration test usage
```

Comment:
Is it possible to move this suppression to the linter config and exclude this file `pkg/testutils/utils.go` there?